bigflow/workflow.py

```diff
@@ -417,3 +420,55 @@ def _parse_runtime_str(runtime: str) -> dt.datetime:
         except ValueError:
             pass
     raise ValueError("Unable to parse 'runtime' %r" % runtime)
+
+
+def load_workflows_in_packages(
+        project_dir: Path,
+        packages: List[str]
+) -> Iterator[Any]:
+    """
+    Imports modules from specified packages and yields elements of type bf.Workflow
+    """
+    if not packages:
+        raise ValueError('At least one package required')
+
+    # TODO move walk_workflows from cli to workflows
+    from bigflow.cli import walk_workflows
+    return (
+        w
+        for p in packages if '.' not in p
+        for w in walk_workflows(project_dir / p)
+    )
+
+
+def verify_workflow(workflow_id: str) -> None:
```

Comment:
If I understand correctly, if I want to validate all workflows, then I need to pass some random string.

Maybe just make the workflow_id optional, and if it's None, then set a random string inside the method, it's easier to understand.

Reply:
AFAIK now it should be `"ALL"` or `""` (empty string)  to validate all workflows. `None` also works (despite the typehint), because `bigflow.commons.is_workflow_id_specified` also implicitly check for `None`.

BTW magic `"ALL"` is replaced by `None` in `cli.py` https://github.com/allegro/bigflow/blob/233fe90c039cec72f44601cc7a6f578926b884e6/bigflow/cli.py#L570
so `verify_workflow` is never called with an empty string nor the `"ALL"`

Reply:
Good catch, this was not intended ofc. Will fix during the current refactoring

Reply:
BTW. the same with "--start-time", it is replaced by `datetime.now().strftime("%Y-%m-%d %H:00:00")` in cli.py
This is what i meant when was saying "NOW and ALL are parts of cli". Sorry, it was not written clearly.